lib/modules/platform/bitbucket/index.ts

```diff
@@ -309,6 +310,33 @@ export async function findPr({
   if (pr) {
     logger.debug(`Found PR #${pr.number}`);
   }
+
+  /**
+   * Bitbucket doesn't support renaming or reopening declined PRs.
+   * Instead, we have to use comment-driven signals.
+   */
+  if (pr && pr.state === 'closed') {
+    const REOPEN_PR_COMMENT_KEYWORD = 'reopen!';
+
+    const reviewers = await effectiveDefaultReviewers(config.repository);
+    const reviewerUuids = reviewers.map((reviewer) => reviewer.uuid);
```

Comment:
How about fetching the reviewers only if comments are found? i.e. instead of fetching them eagerly

Reply:
Great feedback @rarkins .  Incorporated with latest push